components/Drop/DropProgress.tsx

```diff
@@ -0,0 +1,77 @@
+import { Flex, Progress, SimpleGrid, Text } from '@chakra-ui/react'
+import BigNumber from 'bignumber.js'
+import useTranslation from 'next-translate/useTranslation'
+import numbro from 'numbro'
+import { FC, useMemo } from 'react'
+
+type Props = {
+  drops: {
+    minted: string
+    supply: string | null
+  }[]
+}
+
+const DropProgress: FC<Props> = ({ drops }) => {
+  const { t } = useTranslation('components')
+
+  const totalMinted = useMemo(
+    () =>
+      drops.reduce((acc, drop) => {
+        return acc + BigNumber(drop.minted).toNumber()
+      }, 0),
+    [drops],
+  )
+
+  const totalSupply = useMemo(
+    () =>
+      drops.reduce((acc, drop) => {
+        if (drop.supply === null) return Infinity
+        return acc + BigNumber(drop.supply).toNumber()
+      }, 0),
+    [drops],
+  )
+
+  const mintPercentage = useMemo(() => {
+    if (!totalSupply) return
+    return new BigNumber(totalMinted)
+      .div(BigNumber(totalSupply))
+      .multipliedBy(100)
+      .toNumber()
+  }, [totalMinted, totalSupply])
```

Comment:
To not have to use bignumber.js, you can do multiply by 10000 (basis points), calculate the ration, convert to number (safe because it's a ratio), and then divide by 100 to get the percent:
```suggestion
  const mintPercentage = useMemo(() => {
    if (!totalSupply) return
    return BigNumber.from(totalMinted).mul(10000).div(totalSupply).toNumber() / 100
  }, [totalMinted, totalSupply])
```

Reply:
[Remove bignumber.js imports](https://github.com/liteflow-labs/starter-kit/pull/359/commits/80d377f017d0d7ebae6617c8c386a018de3c7e6a)